nrm_django/nrm_app/templates/home.html

```diff
@@ -1,6 +1,65 @@
+{% extends 'base.html' %}
+{% load static %}
 
+If we have a username here, then we're accessing the db and seeing the currently logged-in user:
```

Comment:
If we still want this content to appear on the page, then it has to go inside the `{% block main %}`. In this place in the template, it never gets onto the rendered page.

Reply:
I meant to take it out. One minute...